backend/src/packages/auth/auth.service.ts

```diff
@@ -1,5 +1,10 @@
+import * as jose from 'jose';
```

Comment:
```suggestion
```
create a token package for it

avoid `* as` import

Reply:
What do you mean with "create package"? trying to find something about it half of night and still don`t have understanding what to do.

Reply:
I thing i got it. No will try to fix